.github/workflows/comment-changes.yml

```diff
@@ -0,0 +1,34 @@
+name: comment-changes
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  aeruginous:
+    name: aeruginous
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.5.2
+        with:
+          fetch-depth: 0
+
+      - run: |
+          rustup update
+          cargo install aeruginous
+          aeruginous comment-changes -d ::= -o changelog.d/ -k -n 1
+
+      - uses: peter-evans/create-pull-request@v5.0.1
```

Comment:
I think it'll work without having to have a pull request per fragment. since this actions grapes changes in the repo. We can try it in see how well it'll work and make adjustments along the way